app/policies/evss_policy.rb

```diff
@@ -12,20 +12,20 @@ def access_form526?
   private
 
   def evss_attrs?
-    user.edipi.present? && user.ssn.present? && user.participant_id.present?
+    user.present? && user.edipi.present? && user.ssn.present? && user.participant_id.present?
   end
 
   def form526_attrs?
-    user.birls_id.present? && evss_attrs?
+    user.present? && user.birls_id.present? && evss_attrs?
```

Comment:
So as to not duplicate the `user.present?` check:
```suggestion
    evss_attrs? && user.birls_id.present?
```

Reply:
Oops, good call!